CSS/style.css

```diff
@@ -0,0 +1,103 @@
+@import url("/CSS/utils.css");
+
+.logo-link {
+  font-size: 1.5rem;
+}
+.rules-link {
+  font-size: 1rem;
+}
+.rules-link:hover,
+.profile-icon:hover {
+  color: var(--Primary_Color);
+}
+
+/* Intro section */
+.app-intro {
+  padding: 1rem;
+  display: flex;
+  gap: 10px;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+.intro-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.intro-header,
+.intro-quote,
+.intro-greeting {
+  font-weight: 400;
+  opacity: 0.9;
+  margin: 0;
+  font-size: 1.5rem;
+}
+.intro-greeting {
+  font-size: 1rem;
+}
+.intro-image {
+  max-width: 28rem;
+  width: 100%;
+}
+
+.btn {
+  font-size: 1rem;
+  background-color: transparent;
+  border-radius: 5px;
+  margin-top: 2rem;
+  transition: 0.3s;
+}
+
+/* Quiz section */
+
+.quiz-wrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 10px;
+  padding: 1rem;
+}
+.quiz-container {
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: center;
```

Comment:
You are repeating these styles in many places so consider making a class for them as well.
```
display:flex;
align-items:center;
```

Reply:
Got it.